src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/mainscreen/view/MainScreenView.kt

```diff
@@ -0,0 +1,127 @@
+                            when(it.type) {
+                                NavBoxType.PROJECT -> {
+                                    majorLabelProperty.bind(viewModel.selectedProjectName)
+                                    minorLabelProperty.bind(viewModel.selectedProjectLanguage)
+                                    visibleProperty().bind(viewModel.selectedProjectProperty.booleanBinding { it != null })
+                                }
+                                NavBoxType.CHAPTER -> {
+                                    titleProperty.bind(viewModel.selectedCollectionTitle)
+                                    bodyTextProperty.bind(viewModel.selectedCollectionBody)
+                                    visibleProperty().bind(viewModel.selectedCollectionProperty.booleanBinding { it != null })
+                                }
+                                NavBoxType.CHUNK -> {
+                                    titleProperty.bind(viewModel.selectedContentTitle)
+                                    bodyTextProperty.bind(viewModel.selectedContentBody)
+                                    visibleProperty().bind(viewModel.selectedContentProperty.booleanBinding { it != null })
+                                }
+                            }
+                        }
+                    }
+                }
+                navButton {
+                    text = messages["back"]
+                    graphic = AppStyles.backIcon()
+                    action {
+                        navigateBack()
+                    }
+                }
+            })
+            fragmentStage = anchorpane {
+                hgrow = Priority.ALWAYS
+                vgrow = Priority.ALWAYS
+                add(listmenu {
+                    orientation = Orientation.HORIZONTAL
+                    item(messages["home"], MaterialIconView(MaterialIcon.HOME, "20px"))
+                    item(messages["profile"], MaterialIconView(MaterialIcon.PERSON, "20px"))
+                    item(messages["settings"], MaterialIconView(MaterialIcon.SETTINGS, "20px"))
+
+                    anchorpaneConstraints {
+                        topAnchor = 0
+                        rightAnchor = 0
+                    }
+                })
+                hbox {
+                    anchorpaneConstraints {
+                        topAnchor = 50
+                        leftAnchor = 50
+                        rightAnchor = 0
+                        bottomAnchor = 0
+                    }
+                    hgrow = Priority.ALWAYS
+                    vgrow = Priority.ALWAYS
+                    activeFragment.dock<ProjectGridView>()
+                    ProjectGridView().apply {
+                        viewModel.selectedProjectProperty.bindBidirectional(activeProject)
+                    }
+                    add(activeFragment)
+                }
+            }
+        }
+    }
+
+    private fun navigateBack() {
+
```

Comment:
Don't change this if it's easier to read this way, but some of these boolean predicates aren't needed:
1. Can remove `viewModel.selectedContentProperty.value == null` from the second `if`, since it's really an `elsif` and you already tested the opposite.
2. Can remove *both* predicates from the third `if`, and just make it an `else`.

Bonus: can also factor the three `activeFragment.navigateBack()`s out and place it outside the whole if/else thing.  (And once you do *that*, then you can even kill the entire third block since it'll be empty.)